indico/MaKaC/webinterface/rh/conferenceDisplay.py

```diff
@@ -69,21 +70,21 @@ def _checkProtection(self):
 
 class MeetingRendererMixin:
     def render_meeting_page(self, conf, view, hide_frame=False):
-        evt_type = conf.getType()
+        evt_type = conf.as_event.type_
         view = view or conf.as_event.theme
 
         # if the current view is invalid, use the default
         if view not in theme_settings.themes:
-            view = theme_settings.defaults[evt_type]
+            view = theme_settings.defaults[evt_type.name]
```

Comment:
I think there are quite a few more legacy places  where it may use simple_event - probably easier to remove once we eradicate `simple_event` everywhere